cl/assets/static-global/css/override.css

```diff
@@ -1709,3 +1709,17 @@ rect.series-segment {
 .htmx-request.htmx-hidden-indicator {
     display:inline;
 }
+
+.turbo-progress-bar {
+  position: fixed;
+  display: block;
+  top: 0;
+  left: 0;
+  height: 3px;
+  background: #B53C2C;
+  z-index: 2147483647;
```

Comment:
😄 I'm sorry to disappoint you, but no, it's not. 

To ensure the progress bar always appears on top of other HTML elements, we need to set its `z-index` to a very high value. The maximum value for z-index is `2147483647`, which is the largest signed 32-bit integer. By setting the progress bar's z-index to this value, it will always be the topmost element on the page, regardless of the z-index values of other elements.